modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts

```diff
@@ -102,14 +108,14 @@ export class EddsaUtils extends baseTSSUtils<KeyShare> {
    * @param userKeyShare - User's TSS Keyshare
    * @param backupKeyShare - Backup's TSS Keyshare
    * @param bitgoKeychain - previously created BitGo keychain; must be compatible with user and backup key shares
-   * @param passphrase - wallet passphrase used to encrypt user's signing materials
+   * @param [passphrase] - optional wallet passphrase used to encrypt user's signing materials
```

Comment:
Needs more information about when the passphrase is optional.

Reply:
Added to the description of the method